chunky/src/java/se/llbit/chunky/ui/RenderCanvasFx.java

```diff
@@ -57,6 +58,16 @@
   private static final WritablePixelFormat<IntBuffer> PIXEL_FORMAT =
       PixelFormat.getIntArgbInstance();
 
+  // This sets the maximum "Render Preview" canvas size to 4k by 4k,
+  // for sizes any larger cause a strange crash in JavaFX.
+  // Currently, this only shows the 4k by 4k top-left corner of renders
+  // which are larger than this. Future task: be able to pan around on
+  // the Render Preview canvas. (Kind-of exists, but again, loading the
+  // whole image leads to mysterious crashes.)
+  // Oh, and also, the crash happens at different minimum canvas sizes
+  // unique to each computer.
+  private static final int MAX_CANVAS_SIZE = 4096;
```

Comment:
This is fine and shouldn't block us from merging this PR. Currently Chunky just crashes, so this is far better! :+1: